cognitedata/reveal
# feat(react-components): add pagination support for styling hook allowing styling more than 1000 nodes

#### Type of change

![Feat](https://img.shields.io/badge/Type-Feat-green)  

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/REV-972

## Description :pencil:

This PR fixes an issue with styling hook where it was not styling more than 1000 nodes by externalId because of lacking pagination support in mapping fetching hook.

## How has this been tested? :mag:
In ModelsStyling example from storybook.



## Test instructions :information_source:

1. Go to ModelsStyling story when on `3dtest` project
2. Trigger different styling modes and see


## Checklist :ballot_box_with_check:
